mne/tests/test_epochs.py

```diff
@@ -1255,4 +1255,33 @@ def test_array_epochs():
     assert_equal(len(events), len(epochs.selection))
 
 
+def test_dataframe_time_indexing():
+    '''Make sure that conversion to dataframe
+    doesn't duplicate time indices due to rounding
+    error + floating point precision'''
+    # Synthesize data
+    data = np.random.randn(2, 10000)
+    data_event = np.zeros(data.shape[1])
+    data_event[[2, 2000]] = 1  # Insert some events
+    data_raw = np.vstack([data, data_event])
+
+    # For the Info object
+    ch_types = ['eeg']*data_raw.shape[0]
```

Comment:
We follow the convention that arithmetic operators should have whitespace between them so `['eeg'] * data_raw.shape[0]` (even though `scipy` and other packages have abandoned that standard)